bundles/org.openhab.binding.renault/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -69,26 +78,88 @@
 				<description>Interval the car is polled in minutes.</description>
 				<default>10</default>
 			</parameter>
+			<parameter name="updateDelay" type="integer" unit="s" min="5" max="120">
+				<label>Update Delay</label>
+				<description>How long to wait for commands to reach car and update to server in seconds.</description>
+				<default>30</default>
+			</parameter>
 		</config-description>
 	</thing-type>
 
-	<!-- Sample Channel Type -->
-	<channel-type id="hvacstatus">
-		<item-type>Switch</item-type>
-		<label>HVAC Status</label>
+	<channel-type id="batteryavailableenergy">
+		<item-type>Number:Energy</item-type>
+		<label>Battery Energy Available</label>
+		<state pattern="%.1f kWh" readOnly="true"></state>
+	</channel-type>
+	<channel-type id="plugstatus">
+		<item-type>String</item-type>
+		<label>Plug Status</label>
+		<description>Status of charging plug.</description>
 		<state readOnly="true"></state>
```

Comment:
Please add state options to map your possible values to a proper label for UI, a label that could even be translated later in different languages.

Reply:
done with enums.